Sources/DistributedActors/Serialization/Serialization+Settings.swift

```diff
@@ -187,15 +177,15 @@ public extension Serialization.Settings {
             "Specialized serializerID MUST exactly `1` or be `> 16`, since IDs until 16 are reserved for general purpose serializers"
         )
         // FIXME: THIS IS A WORKAROUND UNTIL WE CAN GET MANGLED NAMES https://github.com/apple/swift/pull/30318
-        let hint = hintOverride ?? _typeName(type) // FIXME: _mangledTypeName https://github.com/apple/swift/pull/30318
+        let hint = hintOverride ?? _mangledTypeName(type)
```

Comment:
Probably same as before, so 

```suggestion
        let hint = hintOverride ?? _mangledTypeName(type) ?? _typeName(type)
```

?